tooling/api/package.json

```diff
@@ -117,6 +58,7 @@
     "eslint-plugin-node": "11.1.0",
     "eslint-plugin-promise": "4.3.1",
     "eslint-plugin-security": "1.4.0",
+    "fs-extra": "^9.1.0",
```

Comment:
can you pin this dep while we're here :P

Reply:
sure